SourceX/dx.cpp

```diff
@@ -413,36 +420,37 @@ void prepareSpellColors()
 	//SPL_HEALOTHER = 0x22,
 	//SPL_FLARE = 0x23,
 	spellColors[SPL_BONESPIRIT] = green;
-
 }
 
-void prepareFPS(){
+void prepareFPS()
+{
 	if (!frameflag)
 		return;
 	SDL_Surface *fpsVision = SDL_CreateRGBSurfaceWithFormat(0, 50, 50, SDL_BITSPERPIXEL(format), format);
 	if (fpsVision == NULL)
 		ErrSdl();
-	if(SDL_SetSurfaceBlendMode(fpsVision, SDL_BLENDMODE_ADD) < 0)
+	if (SDL_SetSurfaceBlendMode(fpsVision, SDL_BLENDMODE_ADD) < 0)
 		ErrSdl();
-	if(SDL_FillRect(fpsVision, NULL, SDL_MapRGB(fpsVision->format, 255, 255, 255)) < 0)
+	if (SDL_FillRect(fpsVision, NULL, SDL_MapRGB(fpsVision->format, 255, 255, 255)) < 0)
 		ErrSdl();
 	fpsTex = SDL_CreateTextureFromSurface(renderer, fpsVision);
 	if (fpsTex == NULL)
 		ErrSdl();
 	SDL_FreeSurface(fpsVision);
-	if(SDL_SetTextureBlendMode(fpsTex, SDL_BLENDMODE_ADD) < 0)
+	if (SDL_SetTextureBlendMode(fpsTex, SDL_BLENDMODE_ADD) < 0)

```

Comment:
This shouldn't be necessary now that you sorted out the ui

Reply:
it's optional, if someone wants to see fps, I'd expect to see it all t he time :P deactivates when fps display is turned off

Reply:
But isn't it rendered on top of the game now like the rest of the ui?

Reply:
lol true